en/releases/1.14.md

```diff
@@ -1,4 +1,11 @@
-## PX4-Autopilot v1.14 Release Notes
+# PX4-Autopilot v1.14 Release Notes
+
+## Upgrading to PX4 v1.14
```

Comment:
The biggest change here will be the fact that if you rely on a custom mixer you will have to recreate your settings.

Further, we dumped a whole lot of frames definitions. I'm not certain, but I "expect" that if you're using a frame that is not defined in PX4 v1.14 you also have to run geometry setup - right?  Basically you're ONLY OK to not run the actuator setup if you used an airframe that existed previously. 

We should say that and forward link to the actuator section below in the release notes.

Reply:
Oh right, we should def mention that we removed certain airframes, and alert that for custom mixers, they will need to set up again

Reply:
Cool. I want to cut a v1.14 version of the docs soon. Any ETA on doing these updates?

Reply:
I added the mention of airframes below. I just ran a test. IMO **everyone** needs to at least check that their outputs are mapped correctly - for example Holybro X2 airframe defaults to DShot ESC on AUX ports, whereas I am pretty sure the kit ships with PWM ESC and the old instructions had them on MAIN outputs. It can do no harm!